### PR TITLE
update to match exposed uswds pageheader

### PR DESCRIPTION
**Pull request by sandrahoang686**
## Available PR templates

- [Default](?expand=1&template=default.md)
- [Version Release](?expand=1&template=version_release.md)
- _Alternatively delete and start empty_

### CODE REVIEW

**Comment by sandrahoang686** [comment]
NOTE: Still need to build library package and update version to see preview....

File: app/components/nav.tsx [hunk 1]
```diff
@@ -34,30 +38,16 @@ export function Navbar() {
...
-            <PageHeader
-              mainNavItems={navItems}
-              subNavItems={[]}
-              logo={Logo}
-              linkProperties={linkProps}
-            />
-          </div>
-        </nav>
-      </div>
+      <nav id='nav'>
```
[the remark this answers is not in the document]
Comment (sandrahoang686) [reply app/components/nav.tsx:43]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav - it came with the template nextJs setup, could probably just remove it

**Comment by dzole0311** [comment]
Note, we need to still add the About page and the contact us modal. I added this as a todo item here: https://github.com/NASA-IMPACT/veda-ui/issues/1282

File: app/components/nav.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (AliceR) [reply app/components/nav.tsx:52]: I think we should remove the unnecessary elements, `<nav>` and `<aside>`, as we have those implemented with the `<PageHeader>` component itself.

**Review by AliceR — APPROVED** [review]
Just one comment about the DOM tree (see above); the preview looks good 👍

**Pull request merged**